code/exercises/knapsack/knapsack.scm

```diff
@@ -0,0 +1,7 @@
+(import (rnrs (6)))
+
+(load "test.scm")
+
+(define (knapsack capacity weights vlues)
```

Comment:
ugh

Reply:
I was thinking of adding a linting pass making sure each function in the student sub is in the example solution and that the arguments match

Reply:
Sure, that sounds pretty cool.